components/equipment/EquipmentSection.tsx

```diff
@@ -1,34 +1,95 @@
-import React from "react";
+import React, { useState } from "react";
 import { Equipment } from "../../types/hero";
 
-export function EquipmentSection({
-  equipment,
-  isExclusive,
-}: {
-  equipment: Equipment;
-  isExclusive: boolean;
-}) {
+export function EquipmentSection({ equipment }: { equipment: Equipment }) {
   return (
     <>
       <div className="grid grid-cols-12 items-center mt-2 mb-2">
-        <div className="col-span-12 text-center sm:col-span-1">
-          <div className="mb-3">
-            <img
-              src={"/equipment/" + equipment.name + ".png"}
-              className="inline"
-              width={70}
-              height={70}
-            ></img>
-          </div>
+        <div className="col-span-12 text-center sm:col-span-1 flex items-center justify-center mr-3">
+          <img
+            src={"/equipment/" + equipment.name + ".png"}
+            className="inline"
+          />
         </div>
         <div className="col-span-12 text-center sm:col-span-11 sm:text-left">
-          <p className="text-2xl">
-            {isExclusive && "Exclusive Equipment: "}
-            {equipment.name}
-          </p>
-          <p className="whitespace-pre-line">{equipment.effect}</p>
+          <p className="text-2xl mb-2">{equipment.name}</p>
+          <EffectList equipment={equipment} />
         </div>
       </div>
     </>
   );
 }
+
+function EffectList({ equipment }: { equipment: Equipment }) {
+  const [currentKey, setCurrentKey] = useState("lvl50");
+  const [text, setText] = useState(equipment.effect.lvl50);
+  const [stat1, setStat1] = useState(equipment.stat1?.lvl50);
+  const [stat2, setStat2] = useState(equipment.stat2?.lvl50);
```

Comment:
These 4 different state objects would be best served as a single object as they are only changed together

Reply:
fixed